### PR TITLE
MEPTS-356

### PR DESCRIPTION
**Pull request by cfaife**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TxMlQueries.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public static String getPatientsWhoMissedAppointment(
             + "			p.voided = 0 AND e.voided = 0 AND o.voided = 0"
             + "			AND e.encounter_type = %d "
             + "			AND o.concept_id = %d"
-            + "			AND e.encounter_datetime <=:endDate AND e.location_id=:location "
+            + "			AND o.value_datetime <=:endDate AND e.location_id=:location "
```
Comment (ynurmahomed) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TxMlQueries.java:63]: Is this change intended?

Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TxMlQueries.java:63]: yes according with the spec it is

Comment (ynurmahomed) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TxMlQueries.java:63]: Please update pull request title with the ticket reference.

Comment (psmatsinhe) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TxMlQueries.java:63]: Is this related to any ticket? @cfaife

**Review by psmatsinhe — APPROVED** [review]
This looks good
